packages/ui/Routable/Link.js

```diff
@@ -36,6 +95,42 @@ const LinkBase = kind({
 	}
 });
 
+/**
+ * A higher-order component that adds support to a component to handle navigates actions.
+ * When using Linked component you must give the `path` property. And the component should forward `onClick` event.
```

Comment:
About the second line, `path` is required from `Link`, not this hoc so I think we could remove it.
Do you know why we should forward `onClick` event? I think it's somewhat beyond the description of this hoc.  🤔 
```suggestion
 * A higher-order component adds support to a component to handle `navigate` from [`Routable`]{@link ui/Routable.Routable}.
```

Reply:
I've added a description of why it should forward the `onClick` event in association with `{navigate: 'onClick'}`.
Thank you for the review :)